server/src/main/java/org/opensearch/common/util/FeatureFlags.java

```diff
@@ -56,10 +56,10 @@ public class FeatureFlags {
     public static final String DATETIME_FORMATTER_CACHING = "opensearch.experimental.optimization.datetime_formatter_caching.enabled";
 
     /**
-     * Gates the functionality of writeable remote index
+     * Gates the functionality of writeable remote index having the capability to move across different tiers
```

Comment:
`.. functionality of writeable remote index..` ---> `.. functionality of remote index...`